platformLibs/src/platform/osx/System.def.disabled

```diff
@@ -0,0 +1,6 @@
+language = Objective-C
+package = platform.System
+modules = System
+compilerOpts = -framework System
+linkerOpts = -framework System
+#Disabled: Swift-only framework
```

Comment:
Not sure about this: the framework doesn't contain any headers or Swift modules.

Reply:
At the same time, it _will_ be a Swift-only framework starting from macOS 11 https://developer.apple.com/documentation/system?language=objc.